### PR TITLE
Support Inputs from TileDB Prompt User Options Jupyterlab Extension

### PR DESCRIPTION
**Pull request by nguyenv**

### CODE REVIEW

**Comment by nguyenv** [comment]
For support with https://github.com/TileDB-Inc/TileDB-Cloud-Jupyter-Prompt-Options

File: tiledbcontents/tiledbcontents.py [hunk 1]
```diff
@@ -312,9 +312,12 @@ def _save_notebook_tiledb(self, model, uri):
...
 
         final_name = self._write_bytes_to_array(
-            uri, file_contents, model.get("mimetype"), model.get("format"), "notebook"
+            uri, file_contents, model.get("mimetype"), model.get("format"), "notebook", 
+            model["s3_prefix"] if "s3_prefix" in model else None, 
+            model["s3_credentials"] if "s3_credentials" in model else None, 
+            "name" in model,
         )
-
+
```
Comment (Shelnutt2) [inline tiledbcontents/tiledbcontents.py:320]: Can you run black to format the file?

File: tiledbcontents/tiledbcontents.py [hunk 2]
```diff
@@ -477,6 +540,10 @@ def _create_array(self, uri, retry=0):
 
             return tiledb_uri, array_name
         except tiledb.TileDBError as e:
+            with open("/tmp/output", "a") as f:
```
Comment (Shelnutt2) [inline tiledbcontents/tiledbcontents.py:567]: Remove debug code?

File: tiledbcontents/tiledbcontents.py [hunk 3]
```diff
@@ -415,6 +479,15 @@ def _create_array(self, uri, retry=0):
                 ctx=tiledb_create_context,
             )
 
+            parts = uri.split("/")
```
Comment (Shelnutt2) [inline tiledbcontents/tiledbcontents.py:482]: I think `parts`, `parts_len` and `namespace` are also computed above at line 423?

**Review by Shelnutt2 — APPROVED** [review]
Excellent work!

**Review by antalakas — APPROVED** [review]

**Pull request merged**